ksql-rest-app/src/test/java/io/confluent/ksql/rest/entity/KafkaTopicsListExtendedTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import static org.easymock.EasyMock.expect;
```

Comment:
nit: for new tests we prefer Mockito to EasyMock

Reply:
Thanks! I'll update the tests to match this and the comment below.